src/main/java/com/redhat/devtools/lsp4ij/LanguageServersRegistry.java

```diff
@@ -543,6 +543,24 @@ public boolean isFileSupported(@Nullable VirtualFile file, @NotNull Project proj
         return false;
     }
 
+    /**
+     * Returns true if the file and optional language are supported by a language server and false otherwise.
+     *
+     * @param virtualFile the file
+     * @param language    the language
+     * @return true if the language of the file is supported by a language server and false otherwise.
+     */
+    public boolean isFileSupported(@Nullable VirtualFile virtualFile, @Nullable Language language) {
+        if ((virtualFile == null) || !virtualFile.isInLocalFileSystem()) {
```

Comment:
This check with in local has changed, olease synchronize with the new version.

Reply:
> This check with in local has changed, olease synchronize with the new version.

Apologies but I don't follow. I'm showing as fully up-to-date with the `main` branch right now.